app/src/main/java/com/github/se/polyfit/ui/components/GenericScreen.kt

```diff
@@ -0,0 +1,17 @@
+package com.github.se.polyfit.ui.components
+
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import com.github.se.polyfit.ui.screen.OverviewScreeen
+
+@Composable
+fun GenericScreen(
+    navController: NavHostController,
+) {
+
+  Scaffold(
+      topBar = { AppTitle() },
+      bottomBar = { BottomNavigationBar(navController = navController) },
```

Comment:
navController should never been passed as argument, is preferable to pass a function of the navController and i thought the navigation was implemented as decorator and maybe should be used?

Reply:
Okay i passed it as a function and the navigation implemented is in use